.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report bugs or errors
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```

Comment:
I think OS is helpful (although iOS is not a great example), but maybe some better info would be:
```md
- OS: [e.g. Ubuntu 22.04, MacOS 14.5]
- python version [e.g. 3.10]
- R version [e.g. 4.4.0]
- flepiMoP branch/commit if relevant (`git rev-parse HEAD`)
```

Reply:
Both of your suggestions make sense! Thank you I'll change that